librz/core/cmd/cmd_search_rop.c

```diff
@@ -727,3 +730,389 @@ static void rop_classify(RzCore *core, Sdb *db, RzList /*<char *>*/ *ropList, co
 
 	free(str);
 }
+
+RZ_API void rz_rop_constraint_free(void *data) {
+	RzRopConstraint *constraint = data;
+	if (constraint) {
```

Comment:
In such cases you could use the "return early" pattern and reverse the condition, it will make the code slightly easier to read.